hooks/pre/12-clear_pulp_content.rb

```diff
@@ -0,0 +1,73 @@
+require 'fileutils'
+
+def clear_pulp_content
+  content_dir = '/var/lib/pulp/content'
+  if File.directory?(content_dir)
+    logger.debug "Removing Pulp content inside \'#{content_dir}\'."
+    FileUtils.rm_rf(Dir.glob(File.join(content_dir, '*')))
+    logger.debug "Removed Pulp content inside '\#{content_dir}\', now resetting Pulp Repo importers."
+    reset_repo_importers
+    logger.info 'Pulp content successfully removed.'
+  else
+    logger.warn "Pulp content directory not present at \'#{content_dir}\'."
+  end
+end
+
+def remote_host?(hostname)
+  !['localhost', '127.0.0.1', `hostname`.strip].include?(hostname)
+end
+
+def load_mongo_config
+  seeds = param_value('katello', 'pulp_db_seeds')
+  seed = seeds.split(',').first
+  host, port = seed.split(':') if seed
+  {
+    host: host || 'localhost',
+    port: port || '27017',
+    database: param_value('katello', 'pulp_db_name') || 'pulp_database',
+    username: param_value('katello', 'pulp_db_username'),
+    password: param_value('katello', 'pulp_db_password'),
+    ssl: param_value('katello', 'pulp_db_ssl') || false,
+    ca_path: param_value('katello', 'pulp_db_ca_path'),
+    ssl_certfile: param_value('katello', 'pulp_db_ssl_certfile'),
+  }
+end
+
+def reset_repo_importers
+  mongo_config = load_mongo_config
+  if remote_host?(mongo_config[:host])
+    reset_repo_importers_remote_mongo(mongo_config)
+  else
+    start_services(['rh-mongodb34-mongod'])
+    execute("mongo pulp_database --eval 'db.repo_importers.update({"scratchpad": {$ne: null}}, {$set: {"scratchpad.repomd_revision": null}}, {"multi":true})'")
```

Comment:
You need to escape here. Previously it was in backticks but now that it's in quotes, you should. Given there are no variables, I'd suggest
```suggestion
    execute('mongo pulp_database --eval \'db.repo_importers.update({"scratchpad": {$ne: null}}, {$set: {"scratchpad.repomd_revision": null}}, {"multi":true})\'')
```